lib/logflare/sources.ex

```diff
@@ -341,15 +341,13 @@ defmodule Logflare.Sources do
     %{source | metrics: new_metrics}
   end
 
-  def valid_source_token_param?(string) when is_binary(string) do
-    case String.length(string) === 36 && Ecto.UUID.cast(string) do
+  def valid_source_token_param?(string) do
```

Comment:
this uuid check should use this Ecto.UUID.dump instead, see here
https://github.com/Logflare/logflare/pull/1617